types/Circus.js

```diff
@@ -9,7 +9,7 @@
 
 export type DoneFn = (reason?: string | Error) => void;
 export type BlockFn = () => void;
-export type BlockName = string | Function;
+export type BlockName = string;
```

Comment:
@aaronabramov whenever you find the time, does this make sense? We do not support just a function in `describe`.

![image](https://user-images.githubusercontent.com/1404810/47255172-185be700-d46d-11e8-920f-a1144833dbbf.png)